mac/sound.cpp

```diff
@@ -970,8 +970,8 @@ OSStatus CSound::callbackIO ( AudioDeviceID          inDevice,
             for ( int i = 0; i < iCoreAudioBufferSizeMono; i++ )
             {
                 // copy left and right channels separately
-                pSound->vecsTmpAudioSndCrdStereo[2 * i]     = (short) ( pLeftData[iNumChanPerFrameLeft * i + iSelInInterlChLeft] * _MAXSHORT );
-                pSound->vecsTmpAudioSndCrdStereo[2 * i + 1] = (short) ( pRightData[iNumChanPerFrameRight * i + iSelInInterlChRight] * _MAXSHORT );
+                pSound->vecsTmpAudioSndCrdStereo[2 * i]     = pLeftData[iNumChanPerFrameLeft * i + iSelInInterlChLeft];
```

Comment:
Have you compiled and checked the functionality for the Mac sound interface? Or just changed the code?

Reply:
Just changed the code. I have a Mac so I can probably test there.

Reply:
Compile and test OK!